src/LLM/InferencingParams.cs

```diff
@@ -0,0 +1,17 @@
+namespace Spin.SDK.LLM;
+
+public class InferencingParams
+{
+    public uint MaxTokens { get; set; }
```

Comment:
Am I correct that all these properties will default to zero values (and remain that way unless the programmer remembers to set them)?  If so, should we change the defaults to more useful values?  And/or add a constructor that requires some of them to be set by the programmer?

Reply:
that makes sense. I'll check with other SDKs to see which default value we used there

Reply:
Adopted default values from JS SDK (see https://github.com/fermyon/spin-js-sdk/blob/e70f94396ee7d5caa6107c84039903f05ba351d3/src/llm.ts#L98 and following)